bindata/Makefile

```diff
@@ -0,0 +1,27 @@
+# Mgmt
+# Copyright (C) 2013-2017+ James Shubin and the project contributors
+# Written by James Shubin <james@shubin.ca> and the project contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# The bindata target generates go files from any source defined below. To use
+# the files, import "github.com/purpleidea/mgmt/bindata" and use
+# `bytes, err := bindata.Asset("FILEPATH")` where FILEPATH is the path of the
+# original input file relative to `bindata/`. To make the target you will need
+# to install go-bindata using `go get -u github.com/jteeuwen/go-bindata/...`.
+default:
```

Comment:
can we instead have:

1) A .PHONY target called `build` or something
2) A line with `default: build`
3) the name of the file as the target so it refreshes properly, eg:
`COPYING.go: ../COPYING`

4) `build: COPYING.go` so the dep works (users can add other deps here in the future)